website/src/components/markdown/Heading.js

```diff
@@ -4,7 +4,15 @@ import { jsx } from '@emotion/core';
 import { colors, gridSize } from '@arch-ui/theme';
 import { LinkIcon } from '@arch-ui/icons';
 
-import { mq } from '../../utils/media';
+// offset header height for hash links
+const hashLinkOffset = {
+  '::before': {
+    content: '" "',
+    height: 'calc(32px + 60px)',
```

Comment:
Not sure why this is using `calc`, I must've had some dynamic number in there at one point. Happy to leave as is for now.